stdlib/pervasives.md

```diff
@@ -910,7 +910,7 @@ Parameters:
 
 |param|type|description|
 |-----|----|-----------|
-|`0`|`Bool`|The condition to assert|
+|`condition`|`Bool`|The condition to assert|
```

Comment:
@phated It no longer uses the positions, the arguments your talking about are labeled?